documentation/src/main/asciidoc/userguide/chapters/query/spatial/Spatial.adoc

```diff
@@ -88,135 +71,99 @@ relevant section.
 Oracle10g/11g::
 
-There is currently only one Oracle spatial dialect: `OracleSpatial10gDialect` which extends the Hibernate dialect `Oracle10gDialect`.
-This dialect has been tested on both Oracle 10g and Oracle 11g with the `SDO_GEOMETRY` spatial database type.
-+
-This dialect can be configured using the Hibernate property:
+There is currently only support for the `SDO_GEOMETRY` type.
 +
-`hibernate.spatial.connection_finder`:::
-the fully-qualified class name for the implementation of the `ConnectionFinder` to use (see below).
-
-.The `ConnectionFinder` interface
-[NOTE]
-====
 The `SDOGeometryType` requires access to an `OracleConnection` object when converting a geometry to SDO_GEOMETRY.
 In some environments, however, the `OracleConnection` is not available (e.g. because a Java EE container or connection pool proxy wraps the connection object in its own `Connection` implementation).
-A `ConnectionFinder` knows how to retrieve the `OracleConnection` from the wrapper or proxy Connection object that is passed into prepared statements.
-
+A `ConnectionFinder` knows how to retrieve the `OracleConnection` from the wrapper or proxy Connection object that is passed into prepared statements. It can be configured with the `hibernate.spatial.connection_finder` property:
++
 When the passed object is not already an `OracleConnection`, the default implementation will attempt to retrieve the `OracleConnection` by recursive reflection.
 It will search for methods that return `Connection` objects, execute these methods and check the result.
 If the result is of type `OracleConnection` the object is returned.
 Otherwise, it recurses on it.
-
++
 In may cases, this strategy will suffice.
 If not, you can provide your own implementation of this interface on the classpath, and configure it in the `hibernate.spatial.connection_finder` property.
 Note that implementations must be thread-safe and have a default no-args constructor.
-====
-
 SQL Server::
-The dialect `SqlServer2008Dialect` supports the `GEOMETRY` type in SQL Server 2008 and later.
+Currently, The `GEOGRAPHY` type is not currently supported.
 
-[NOTE]
-====
-The `GEOGRAPHY` type is not currently supported.
-====
 
 CockroachDB::
-The dialect `CockroachDB` support the `GEOMETRY` type in CockroachDB v20.2 and later.
-
-[NOTE]
-====
-The `GEOGRAPHY` type is not currently supported.
-====
+The dialect `CockroachDB` support the `GEOMETRY` type in CockroachDB v20.2 and later. The `GEOGRAPHY` type is currently not supported.
 
-GeoDB (H2)::
-The `GeoDBDialect` supports the GeoDB a spatial extension of the H2 in-memory database.
-[NOTE]
-====
-The dialect has been tested with GeoDB version 0.7
-====
+H2GIS::
+The `H2DBDialect` supports the H2GIS, a spatial extension of the H2 in-memory database. This dialect can be used as a replacement for the `GeoDB` dialect that was supported in previous versions. The major difference with `GeoDB` is that the  `GEOGRAPHY` column type is not present in `H2GIS`.
```

Comment:
```suggestion
The `H2Dialect` supports H2GIS, a spatial extension of the H2 in-memory database. This dialect can be used as a replacement for the `GeoDB` dialect that was supported in previous versions. The major difference with `GeoDB` is that the  `GEOGRAPHY` column type is not present in `H2GIS`.
```

Reply:
I think the missing support of geometry type in h2gis might be temporary, see https://github.com/orbisgis/h2gis/issues/864. So we might better tweak verbiage of 'not present' to 'not supported currently'. Otherwise it implies that the decision is by design and be cast in stone.